vpro-shared-monitoring/src/main/java/nl/vpro/monitoring/config/MonitoringConfig.java

```diff
@@ -92,6 +95,15 @@ public PrometheusMeterRegistry globalMeterRegistry() {
                 new DiskSpaceMetrics(new File(folder)).bindTo(registry);
             }
         }
+        if (properties.isMeterLocks()) {
+            Gauge.builder("locks.currentCount", ObjectLockerAdmin.JMX_INSTANCE, ObjectLockerAdmin::getCurrentCount)
```

Comment:
Use lower case by convention zo current.count. Not sure wether "count" provides the strongest semantics here, does it mean: "lock active"?

Reply:
would perhaps be better, but I think its also worth somethng to follow the method names already in use.